### ISSUE TITLE
Translations not retrieve

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#1078]
### Describe the bug

Some of my translations are not retrieved from `{language}/formie.php` file.

This is the case for:

- Dropdown Label
- Dropdown Option Label

Fortunately, I can set them from the general `{language}/site.php` file. But that would be useful to be able to add them from `{language}/formie.php` file like other Formie translations.

### Steps to reproduce

1. Add a Dropdown Label and Dropdown Option Label to the form
2. Try to translate them using `{language}/formie.php` file.

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.2.3

### Plugin version

2.0.9

### Multi-site?

Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.10](https://github.com/verbb/formie/releases/tag/2.0.10)
